extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,429 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="weatherunderground" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- WeatherUnderground Binding -->
+	<thing-type id="weather">
+		<label>Weather Information</label>
+		<description>Provides various weather data from the Weather Underground service</description>
+
+		<channel-groups>
+			<channel-group id="current" typeId="current" />
+			<channel-group id="forecastToday" typeId="forecast" />
```

Comment:
IMHO it would make sense to also give them a label and description _here_, so users can better distinguish them. Otherwise you will see "Weather Forecast" for all of them and can only tell by looking an the ChannelUIDs for which day they actually are.